### PR TITLE
Feature/1860 - Points to raster with rotation

### PR DESCRIPTION
**Pull request by Rurpot**
* Apply rotation to points to raster interpolation
* Fix private route when token expires

### CODE REVIEW

File: src/scenes/shared/rasterData/rasterFromPoints.tsx [hunk 1]
```diff
@@ -116,22 +96,47 @@ const rasterFromPoints = (props: IProps) => {
...
-                        columns={[
-                            {key: 0, value: 'x', text: 'x', type: ECsvColumnType.NUMBER},
-                            {key: 1, value: 'y', text: 'y', type: ECsvColumnType.NUMBER},
-                            {key: 2, value: 'z', text: 'z', type: ECsvColumnType.NUMBER}
-                        ]}
-                        onCancel={() => null}
-                        onSave={handleChangeData}
-                        withoutModal={true}
-                    />
+                        <AdvancedCsvUpload
```
Comment (rabbl) [inline src/scenes/shared/rasterData/rasterFromPoints.tsx:130]: There are tabs?

File: src/scenes/shared/rasterData/rasterFromPoints.tsx [hunk 2]
```diff
@@ -116,22 +96,47 @@ const rasterFromPoints = (props: IProps) => {
         props.onChange(cRaster);
     };
 
+    // TODO: Points are behind contour layer but should be in front of it
```
Comment (rabbl) [inline src/scenes/shared/rasterData/rasterFromPoints.tsx:99]: Link to the issue here

**Review by rabbl — CHANGES_REQUESTED** [review]

File: src/scenes/shared/rasterData/rasterFromPoints.tsx (hunk 2 above)
Comment (Rurpot) [reply src/scenes/shared/rasterData/rasterFromPoints.tsx]: added issue number

**Review by rabbl — APPROVED** [review]

**Review by Rurpot — COMMENTED** [review]
Added issue number to todo

**Pull request merged**